src/NativeScript/CMakeLists.txt

```diff
@@ -195,6 +195,12 @@ set_target_properties(NativeScript PROPERTIES
     "
 )
 
+add_custom_command(TARGET NativeScript
```

Comment:
That was the first thing I did, but I couldn't find out how to pass the relative variables to the other target.